coralproject/talk
# [CORL-2332]: add mutation to update user SSOProfile id

## What does this PR do?

These changes add a mutation to update a user's SSOProfile id. This is for https://vmproduct.atlassian.net/browse/CORL-2332.

## What changes to the GraphQL/Database Schema does this PR introduce?

Adds `updateSSOProfileID` to the GraphQL schema. Adds `updateSSOProfileIDUser` to user.

## How do I test this PR?

You can test this PR by running with the `coral-cli` updates branch `feat/CORL-2332-updating-sso-id` to add a command to call this mutation. You can run the new command there and see that it updates the SSO profile id as expected, and provides appropriate errors when unsuccessful.
 
## How do we deploy this PR?
